content/docs/18-knowledgebase/00-how-to/2-palette-sso-with-adfs.md

```diff
@@ -0,0 +1,222 @@
+---
+title: 'Palette SSO with Microsoft AD FS'
+metaTitle: 'Setting up Palette SSO with Microsoft AD FS'
+metaDescription: 'Setting up Palette SSO with Microsoft AD FS'
+icon: ""
+hideToC: false
+fullWidth: false
+hideToCSidebar: false
+hiddenFromNav: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Palette SSO with Microsoft AD FS
+Single sign-on (SSO) is an authentication method that enables users to log in to multiple applications and websites with one set of credentials. SSO works upon a trust relationship established and maintained between the service provider (SP) and an identity provider (IdP) using certificates. Spectro Cloud Palette supports SSO based on either SAML or OIDC.
+
+This section talks about how we can enable Palette SSO with [Microsoft AD FS](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/development/ad-fs-openid-connect-oauth-concepts) based on OIDC.
+
+<InfoBox>
+
+  Note that you cannot use Microsoft AD FS for SAML based SSO with Palette, since Microsoft AD FS does not support the newer Canonical XML 1.1 standard that Palette employs. Hence you can only use the OIDC based approach for Microsoft AD FS.
+
+</InfoBox>
+
+
+# Prerequisites
+In order to setup OIDC based SSO with Microsoft AD FS, you need to use one of the following versions:
+* Microsoft AD FS 2022 (comes with Windows Server 2022)
+* Microsoft AD FS 2019 (comes with Windows Server 2019)
+* Microsoft AD FS 2016 (comes with Windows Server 2016)
+
+If you need to be able to your AD FS service from outside your corporate network (this is typically the case), you will also need an AD FS Reverse Proxy. Microsoft unfortunately does not provide a tutorial on how to set this up, but you can find a good alternative [here](https://blog.matrixpost.net/set-up-active-directory-federation-services-ad-fs-5-0-adfs-reverse-proxy-part-2/)
+
+
+# Enablement
+## 1 - Create the AD FS Application Group for Palette
+
```

Comment:
@kreeuwijk For the rest of the text, can you add the step to a bullet item please. The reason for this behavior is that if a user is stuck or is having an issue with a step, they can tell us, "I'm on step 8 ...".
You can keep the sub-headings as it helps beak up the information.